controllers/bsl.go

```diff
@@ -47,19 +47,18 @@ func (r *DPAReconciler) ValidateBackupStorageLocations() (bool, error) {
 				return false, fmt.Errorf("no provider specified for one of the backupstoragelocations configured")
 			}
 
-			// TODO: cases might need some updates for IBM/Minio/noobaa
 			switch provider {
-			case AWSProvider, "velero.io/aws":
```

Comment:
I you run `grep -Inr '"velero.io/aws"' .`, you will note only place this is used (aside from test files) is here. Should we remove this and only allow `aws`?

Reply:
> only allow aws

so `velero.io/aws` would not work? cause that's what people who move from upstream velero would expect to use.